docs/pages/reference/nextjs/POST.mdx

```diff
@@ -10,4 +10,19 @@ It handles all the redirecting for you, correctly, based on the `FrameContainer`
 ```ts [./app/frames/route.ts]
 
 export { POST } from "frames.js/next/server";
+```
+
+### With url-space saving redirects
+
+```ts [./app/frames/route.ts]
+import { POST as POSTNext, PreviousFrame } from "frames.js/next/server";
+import { NextRequest, NextResponse } from "next/server";
+
+export function POST(req: NextRequest, res: NextResponse) {
+  const redirectHandler = (prevFrame: PreviousFrame) => {
+    if (prevFrame.postBody?.untrustedData.buttonIndex === 4)
+      return "https://www.framesjs.org";
+  };
+  return POSTNext(req, res, redirectHandler);
+}
 ```
```

Comment:
Should the message be validated here?

Reply:
maybe - or maybe not, not sure a redirect needs validation